### PR TITLE
Bulk move all jobs in us-central1-f to us-west1-b.

### PR DESCRIPTION
**Pull request by amwat**
Follow up to https://github.com/kubernetes/test-infra/pull/10999.

Should mitigate https://github.com/kubernetes/test-infra/issues/11001

/assign @spiffxp 
/cc @BenTheElder @krzyzacy @fejta @cblecker @mariantalla

### CODE REVIEW

File: config/jobs/bazelbuild/rules_k8s/rules_k8s_config.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 presubmits:
   bazelbuild/rules_k8s:
   - name: pull-rules-k8s-e2e
-    cluster: gke_rules-k8s_us-central1-f_testing
+    cluster: gke_rules-k8s_us-west1-b_testing
```
Comment (krzyzacy) [inline config/jobs/bazelbuild/rules_k8s/rules_k8s_config.yaml]: I think we want to keep this

File: config/jobs/kubernetes/sig-gcp/sig-gcp-gce-config.yaml [hunk 2]
```diff
@@ -383,7 +383,7 @@ periodics:
       - --cluster=bootstrap-e2e-gce-mz
       - --env=MULTIZONE=true
       # Where the nodes reside.  Master is in the first one.
-      - --env=E2E_ZONES=us-central1-a us-central1-b us-central1-f
+      - --env=E2E_ZONES=us-central1-a us-central1-b us-west1-b
```
Comment (spiffxp) [inline config/jobs/kubernetes/sig-gcp/sig-gcp-gce-config.yaml]: all of these zones probably need to change

File: config/jobs/kubernetes/sig-gcp/sig-gcp-gke-config.yaml [hunk 3]
```diff
@@ -255,10 +255,10 @@ periodics:
       - --extract=ci/latest
       - --gcp-cloud-sdk=gs://cloud-sdk-testing/ci/staging
       - --gcp-node-image=gci
-      - --gcp-zone=us-central1-f
+      - --gcp-zone=us-west1-b
       - --ginkgo-parallel
       - --gke-environment=test
-      - --gke-node-locations=us-central1-f,us-central1-a,us-central1-b
+      - --gke-node-locations=us-west1-b,us-central1-a,us-central1-b
```
Comment (spiffxp) [inline config/jobs/kubernetes/sig-gcp/sig-gcp-gke-config.yaml]: these probably need to be changed as a group as well

File: config/jobs/kubernetes/sig-network/ingress-gce-e2e.yaml [hunk 4]
```diff
@@ -113,7 +113,7 @@ periodics:
       - --cluster=
       - --env=MULTIZONE=true
       # Where the nodes reside.  Master is in the first one.
-      - --env=E2E_ZONES=us-central1-a us-central1-b us-central1-f
+      - --env=E2E_ZONES=us-central1-a us-central1-b us-west1-b
```
Comment (spiffxp) [inline config/jobs/kubernetes/sig-network/ingress-gce-e2e.yaml]: and here

File: config/jobs/kubernetes/sig-gcp/sig-gcp-gce-config.yaml (hunk 2 above)
Comment (cblecker) [reply config/jobs/kubernetes/sig-gcp/sig-gcp-gce-config.yaml]: Yeah, I don't think we can stretch across regions as these will be different subnetwork objects

Comment (krzyzacy) [reply config/jobs/kubernetes/sig-gcp/sig-gcp-gce-config.yaml]: it's probably fine given only central1-f is affected?

File: config/jobs/bazelbuild/rules_k8s/rules_k8s_config.yaml (hunk 1 above)
Comment (amwat) [reply config/jobs/bazelbuild/rules_k8s/rules_k8s_config.yaml]: hmm, maybe not? if we change the zone the name might be misleading

Comment (amwat) [reply config/jobs/bazelbuild/rules_k8s/rules_k8s_config.yaml]: /shrug hmm, although the cluster name would now be misleading .

File: config/jobs/kubernetes/sig-gcp/sig-gcp-gke-config.yaml (hunk 3 above)
Comment (amwat) [reply config/jobs/kubernetes/sig-gcp/sig-gcp-gke-config.yaml]: hmm, I'm not sure on the details of these multizone jobs, but perhaps I'll revert changing these tests? or should we hope they would work similarly in a different region?

File: config/jobs/bazelbuild/rules_k8s/rules_k8s_config.yaml (hunk 1 above)
Comment (krzyzacy) [reply config/jobs/bazelbuild/rules_k8s/rules_k8s_config.yaml]: I think we have an existing build cluster running on central1-f, and have the cluster config saved as a prow secret, so this job is still going to work..

File: config/jobs/kubernetes/sig-gcp/sig-gcp-gke-config.yaml (hunk 3 above)
Comment (spiffxp) [reply config/jobs/kubernetes/sig-gcp/sig-gcp-gke-config.yaml]: neither am I, no preference which you do, but I would leave them all in a single region

File: config/jobs/bazelbuild/rules_k8s/rules_k8s_config.yaml [hunk 5]
```diff
@@ -13,7 +13,7 @@ presubmits:
         - -c
         - |
           gcloud auth activate-service-account --key-file=/etc/gcp/service-account.json \
-          && gcloud container clusters get-credentials testing --zone=us-central1-f --project=rules-k8s \
+          && gcloud container clusters get-credentials testing --zone=us-west1-b --project=rules-k8s \
```
Comment (krzyzacy) [inline config/jobs/bazelbuild/rules_k8s/rules_k8s_config.yaml]: I think this also need to stay in central1-f since it's an existing cluster...

**Comment by krzyzacy** [comment]
/hold
one more comment (we probably don't want to break other tenants)

File: config/jobs/bazelbuild/rules_k8s/rules_k8s_config.yaml (hunk 1 above)
Comment (fejta) [reply config/jobs/bazelbuild/rules_k8s/rules_k8s_config.yaml]: Sen is correct

File: config/jobs/bazelbuild/rules_k8s/rules_k8s_config.yaml (hunk 5 above)
Comment (fejta) [reply config/jobs/bazelbuild/rules_k8s/rules_k8s_config.yaml]: The only thing we should change is where we create new VMs. Existing resources (like rules-k8s' testing cluster in us-central1-f) are not effected

File: config/jobs/cadvisor/cadvisor.yaml [hunk 6]
```diff
@@ -38,7 +38,7 @@ presubmits:
         - --
         - --deployment=node
         - --gcp-project=cadvisor-e2e
-        - --gcp-zone=us-central1-f
+        - --gcp-zone=us-west1-b
```
Comment (fejta) [inline config/jobs/cadvisor/cadvisor.yaml]: I think we should just change the kubernetes/kubernetes jobs

**Comment by spiffxp** [comment]
wat
```
# cd .; git clone https://go.googlesource.com/lint /bazel-scratch/.cache/bazel/_bazel_root/05618a594cb3499a4b912817865867cd/external/org_golang_x_lint
Cloning into '/bazel-scratch/.cache/bazel/_bazel_root/05618a594cb3499a4b912817865867cd/external/org_golang_x_lint'...
fatal: git fetch-pack: expected ACK/NAK, got 'error: Access denied to IP 104.154.219.78'
fatal: The remote end hung up unexpectedly
```

**Comment by spiffxp** [comment]
🤞

**Pull request merged**